app/dao/services_dao.py

```diff
@@ -111,7 +111,9 @@ def dao_fetch_live_services_data():
     ).outerjoin(
         User, Service.go_live_user_id == User.id
     ).filter(
-        Service.count_as_live == True  # noqa
+        Service.count_as_live == True,  # noqa
```

Comment:
You can avoid `# noqa` by using `.is_`:
```suggestion
        Service.count_as_live.is_(True),
```

Reply:
ahh, sweet, thank you 💖

Reply:
done 👍

Reply:
Still has the `# noqa` comment 😳

Reply:
```
git commit --amend
git push -f origin only_show_live_services
```

shhhh.... it never happened 😯